File: resolucoes/java/lista01/src/main/java/org/example/Ex023.java
/*Numa promoção exclusiva para o Dia da Mulher, uma loja quer dar descontos 
para todos, mas especialmente para mulheres. Faça um programa que leia nome, 
sexo e o valor das compras do cliente e calcule o preço com desconto. Sabendo 
que:
- Homens ganham 5% de desconto
- Mulheres ganham 13% de desconto */

package org.example;

import java.util.Scanner;

public class Ex023 {
    public static void main(String[] args) {
        Ex023.main();
    }

    public static void main() {
        Scanner sc = new Scanner(System.in);

        System.out.print("Informe o sexo do cliente (M/F): ");
        String sexo = sc.next();

        System.out.print("Informe o valor total das compras: ");
        double valorTotal = sc.nextDouble();

        double desconto;

        if (sexo.equalsIgnoreCase("M")) {
            // Homens ganham 5% de desconto
            desconto = valorTotal * 0.05;
        } else if (sexo.equalsIgnoreCase("F")) {
            // Mulheres ganham 13% de desconto
            desconto = valorTotal * 0.13;
        } else {
            System.out.println("Sexo inválido.");
            sc.close();
            return;
        }

        double valorComDesconto = valorTotal - desconto;

        System.out.printf("Valor total das compras: R$ %.2f\n", valorTotal);
        System.out.printf("Desconto: R$ %.2f\n", desconto);
        System.out.printf("Valor com desconto: R$ %.2f\n", valorComDesconto);

        sc.close();
        

    }

}

File: resolucoes/python/lista01/ex023.py
# Numa promoção exclusiva para o Dia da Mulher, uma loja quer dar descontos para todos, mas especialmente para mulheres. Faça um programa que leia nome, sexo e o valor das compras do cliente e calcule o preço com desconto. Sabendo que:
# — Homens ganham 5% de desconto
# — Mulheres ganham 13% de desconto

nome = str(input('Digite seu nome: ')).strip()
sexo = int(input("""Qual seu sexo?
1 - Mulher
2 - Homem
Opção: """))
valor_compras = float(input('Digite o valor das compras: '))

if sexo == 1:
    valor_compras = valor_compras * 0.87
    print(f'Suas compras ficaram no valor de {valor_compras}')

elif sexo == 2:
    valor_compras = valor_compras * 0.95
    print(f'Suas compras ficaram no valor de {valor_compras}')
else:
    print('Selecione uma opção válida para o sexo.')
